src/CoreShop/Component/Core/Order/Processor/CartItemsImmutableProcessor.php

```diff
@@ -68,6 +68,14 @@ public function process(OrderInterface $cart): void
                 $itemDiscount,
                 $context
             );
+
+            /**
+             * https://github.com/coreshop/CoreShop/issues/2572
+             *
+             * Since we are recalculating the items-total, we also need to respect the adjustments and
+             * re-add them to the item total as well.
+            */
```

Comment:
```suggestion
            /*
             * https://github.com/coreshop/CoreShop/issues/2572
             *
             * Since we are recalculating the items-total, we also need to respect the adjustments and
             * re-add them to the item total as well.
             */
```